nautobot/dcim/filters.py

```diff
@@ -882,9 +882,19 @@ class InterfaceFilterSet(
         method="filter_kind",
         label="Kind of interface",
     )
+    parent_interface_id = django_filters.ModelMultipleChoiceFilter(
+        field_name="parent_interface",
+        queryset=Interface.objects.all(),
+        label="Parent interface (ID)",
+    )
+    bridge_id = django_filters.ModelMultipleChoiceFilter(
+        field_name="bridge",
+        queryset=Interface.objects.all(),
```

Comment:
Is there a reason we reverted the filter by `TYPE_BRIDGE` on this?

Reply:
https://github.com/nautobot/nautobot/pull/1651#discussion_r870592839